### ISSUE TITLE
feat(accessibility): Resolve medium accessibility issues

### ISSUE THREAD

**Issue by scarletmerlin123** [OpenCTI-Platform/opencti#17676]
## Use case

Upon scanning the application for section 508 compliance multiple medium accessibility issues were found.

- 30% dialogs no auto-focus
- Filter panels no role=search
- Missing aria-expanded on expand buttons
- Borderline contrast colors
- Tag component contrast not guaranteed
- History views color-only event types
- SearchInput placeholder only
- RichTextField fullscreen button no label
- ResponseDialog refresh button no label

## Current workaround

None
## Proposed solution

Resolve medium accessibility issues to meet WCAG guidelines

## If the feature request is approved, would you be willing to submit a PR?

Yes
